IM_calculation/test/test_intensity_measures/test_intensity_measures.py

```diff
@@ -135,7 +135,7 @@ def test_get_arias_intensity_nd(set_up):
         with open(os.path.join(root_path, INPUT, function + "_g.P"), "rb") as load_file:
             g = pickle.load(load_file)
 
-        test_output = intensity_measures.get_arias_intensity_nd(acc, g, times)
+        test_output = intensity_measures.get_arias_intensity_nd(acc, times)
```

Comment:
You could also remove the retrieval of g from the pickle file above since we aren't using it.

Reply:
I scratched my head to understand the meaning of this test and was shy to make a radical change.. Pickling the value of "g" as if we don't know what it is??  Cool. Will remove the pickle part.

Reply:
I had the same thought...